### PR TITLE
Add version and help flags

### PR DESCRIPTION
**Pull request by dns2utf8**
- Add version flag with docopt, implements #9 
- Increase version to 0.1.1
- Update Cargo.lock

### CODE REVIEW

**Comment by rnestler** [comment]
You probably need to bump the minimal required Rust version from 1.5 to 1.6 which introduced support for `no_std`.

File: src/main.rs [hunk 1]
```diff
@@ -51,6 +78,11 @@ fn main() {
     }
 }
 
+#[derive(Debug, RustcDecodable)]
+struct Args {
+    flag_version: bool,
+}
+
```
Comment (rnestler) [inline src/main.rs]: Could you move this above the `fn main()`? Makes more sense while reading the source.

Comment (dns2utf8) [inline src/main.rs]: Done :+1:

**Comment by dns2utf8** [comment]
I am currently increasing the rust version to see when it builds. 1.5.0 seems to old

**Comment by dns2utf8** [comment]
The builds work from 1.8 on. Would you like me to set it to 1.8 or enable all versions?

**Comment by rnestler** [comment]
I like to test for the minimal supported Rust version in travis. So testing with 1.8, stable and beta should be fine.

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,9 @@
 language: rust
+sudo: false
```
Comment (rnestler) [inline .travis.yml]: Why the `sudo: false`? Isn't this the default?

Comment (dns2utf8) [inline .travis.yml]: hm, from the docs:
```
sudo: required default for repositories enabled before 2015
sudo: false default for repositories enabled in 2015 or later
```
It seems optional, remove then?

Comment (rnestler) [inline .travis.yml]: I would prefer to remove it.

**Comment by dns2utf8** [comment]
:+1:

**Pull request merged**